website/docs/plugin/sdkv2/testing/acceptance-tests/index.mdx

```diff
@@ -267,6 +267,12 @@ By default, there is no logging output when running the `go test` command. Use o
 | `TF_ACC_LOG_PATH` | N/A | Set a file path for all logs during testing. Use `TF_LOG_PATH_MASK` to configure individual log files per test. |
 | `TF_LOG_PATH_MASK` | N/A | Set a file path containing the string `%s`, which is replaced with the test name, to write a separate log file per test. Use `TF_ACC_LOG_PATH` to configure a single log file for all tests. |
 
+If the provider is using the Go standard library [`log` package](https://pkg.go.dev/log) for logging, acceptance testing that utilizes [`helper/resource.ParallelTest()`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource#ParallelTest), and `TF_LOG_PATH_MASK`, the logs associated with each test can output across incorrect files as each new test starts. To resolve this issue, choose one of the following:
```

Comment:
```suggestion
The logs associated with each test can output across incorrect files as each new test starts if the provider is using the Go standard library [`log` package](https://pkg.go.dev/log) for logging, acceptance testing that uses [`helper/resource.ParallelTest()`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource#ParallelTest), and `TF_LOG_PATH_MASK`, . To resolve this issue, choose one of the following approaches:
```

Reply:
😍 Thank you for the explanation and updated verbiage.